docs/semgrep-secrets/getting-started.md

```diff
@@ -0,0 +1,233 @@
+---
+slug: /semgrep-secrets/getting-started
+append_help_link: true
+title: Getting started with Semgrep Secrets
+hide_title: true
+description: Set up secrets scanning in Semgrep to find and rotate valid leaked secrets.
+tags:
+  - Semgrep Secrets
+toc_max_heading_level: 2
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# Getting started with Semgrep Secrets
+
+Detect and triage leaked secrets and credentials by scanning with Semgrep Secrets.
+
+Save time by prioritizing which secrets to rotate based on whether they are valid (active and in-use) or not.
+
+![Semgrep Secrets page](/img/secrets-page.png)
+**_Figure._** Semgrep Secrets page. <br />
+
+This document guides you through the following:
+
+* Enabling Semgrep Secrets
+* Receiving findings in GitHub PR comments
+* Triaging findings
+
+To learn about how Semgrep Secrets detects secrets, see [<i class="fa-regular fa-file-lines"></i> Conceptual overview of Semgrep Secrets](/docs/semgrep-secrets/conceptual-overview).
+
+:::info Feature maturity
+* This feature is in public beta.
+* You or your developers may encounter rough edges. For issues, reach out to [<i class="fa-regular fa-envelope"></i> support@semgrep.com](mailto:support@semgrep.com).
+:::
+
+:::tip Sign-up
+To gain access to this feature, fill out the [<i class="fas fa-external-link fa-xs"></i> Semgrep Secrets Beta](https://get.semgrep.dev/secrets-beta-request.html) form.
+:::
+
+## Supported developer environments and features
+
+This section lists the source code managers (SCMs) that Semgrep Secrets supports.
+
+| Source code manager        | Semgrep Secrets | PR or MR comments for valid secrets findings   |
+| -------------------        | --------------- | ---------------------------------------------- |
+| GitHub - Free and Team     | ✔️               | ✔️                                              |
+| GitHub - Enterprise Cloud  | ✔️               | ✔️                                              |
+| GitHub - Enterprise Server | ❌ (Coming soon)| ❌                                             |
+| GitLab (any plan)          | ❌ (Coming soon)| ❌                                             |
+
+## Enabling Semgrep Secrets
+
+:::info Prerequisites
+* Semgrep Secrets can only be enabled through Semgrep Cloud Platform (SCP). [<i class="fas fa-external-link fa-xs"></i> Create an account](https://semgrep.dev/login) to enable Semgrep Secrets.
+* You have added or onboarded at least one repository to Semgrep Cloud Platform for scanning. See Starting a SAST and SCA scan on a remote repository.
+:::
+
+1. Contact the Semgrep Team by filling out the [<i class="fas fa-external-link fa-xs"></i> Semgrep Secrets Beta](https://get.semgrep.dev/secrets-beta-request.html) form.
```

Comment:
```suggestion
1. Contact the Semgrep team by filling out the [<i class="fas fa-external-link fa-xs"></i> Semgrep Secrets Beta](https://get.semgrep.dev/secrets-beta-request.html) form.
```

Reply:
Committing suggestion - we should probably formalize this somewhere (Prefer Semgrep team over Semgrep Team) - I'll make a Linear task for it.

Reply:
Committing suggestion - we should probably formalize this somewhere (Prefer Semgrep team over Semgrep Team) - I'll make a Linear task for it.

Reply:
Committing suggestion - we should probably formalize this somewhere (Prefer Semgrep team over Semgrep Team) - I'll make a Linear task for it.